kspmanager.py

```diff
@@ -54,7 +54,7 @@ def upload(archive, **options):
             'token': options.get('token').decode('utf8'),
             'name': options.get('name', 'Craft').decode('utf8'),
             'description': options.get('description', '').decode('utf8'),
-            'tags': options.get('tags', 'KSP').decode('utf8'),
+            'tags': 'KSP ' + options.get('tags', '').decode('utf8'),
```

Comment:
this is of no consequence here but beware that you are mixing `str` and `unicode` here.
You could simply `u'KSP ' + options...`